src/components/Users/UsersContainer.jsx

```diff
@@ -1,34 +1,122 @@
 import React from "react";
 import { connect } from "react-redux";
+import * as axios from "axios";
```

Comment:
As far as I remember "import axios from 'axios';" would be enough,  no need to import everything (* = everything) from axios.

Reply:
Ok